packages/rspack/tests/ConfigCase.template.ts

```diff
@@ -121,6 +121,12 @@ export const describeCases = config => {
 								if (!options.target) options.target = "node";
 								if (!options.output) options.output = {};
 								if (!options.output.path) options.output.path = outputDirectory;
+								if (config.newResolver) {
```

Comment:
shouldn't change this configuration, you could just copy configCase/resolve to configCase/new_resolve and add experiments.rspackFuture.newResolver to the webpack.config.js in each case https://github.com/web-infra-dev/rspack/blob/main/packages/rspack/tests/configCases/resolve/fallback/webpack.config.js#L10